study-builder/studies/basil/i18n/ru.conf

```diff
@@ -0,0 +1,884 @@
+{
+  "prequal": {
+    "name": "Prequal",
+    "title": "Заголовок: Prequal",
+    "join_prompt": "Каким образом Вы хотите принимать участие в исследовании?",
+    "join_option_participant": "Я бы хотел принять участие в качестве испытуемого",
+    "join_option_mailing_list": "Я бы хотел подписаться",
+    "join_hint_req": "Пожалуйста, выберите один из вариантов выше",
+  },
+  "autocomplete": {
+    "name": "Picklist Autocomplete",
```

Comment:
Don't feel obligated to translate to Russian, but could you at least at the [ru] prefix for all the strings? Just want to make sure that when we see the app in action through the GUI we don't have to wonder whether the translation correctly by the software or not; if we see text in Russian or with the [ru] prefix we will know there is not bug.